los-deferal-02/Hello-Books
# #166237761- Setup PostgreSQL Database

#### What does this PR do?

- Setup PostgreSQL cloud-hosted database for the project

- Setup databases for testing, development, and production and add the URLs to the project

- Setup configuration for the DB instances

- Write migration scripts for the databases

#### Description of Task to be completed?

- Ensure databases for testing, development, and production are set up and running correctly

- Ensure migration scripts work correctly

#### How should this be manually tested?

- Clone the repo and run the relevant scripts described in the `package.json` file

- Navigate to the relevant database and confirm the success of the script run

#### Any background context you want to provide?
This PR creates PostgreSQL database instances for the project. It creates three databases and provides migration scripts also

#### What are the relevant pivotal tracker stories?
[#166237761](https://www.pivotaltracker.com/story/show/166237761)
